python/CHANGELOG.md

```diff
@@ -5,6 +5,23 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [21.4.0](https://www.github.com/looker-open-source/sdk-codegen/compare/looker_sdk-v21.0.0...looker_sdk-v21.4.0) (2021-04-02)
+
+
+### ⚠ BREAKING CHANGES
+
+* Update the python SDK to expose these methods on the Looker API.
```

Comment:
Interesting how this got parsed. Here's my exact commit message:

```
fix!: python sdk login, login_user, logout methods (#545)

Update the python SDK to expose these methods on the Looker API.

BREAKING CHANGE: sdk.login, sdk.logout, and sdk.login_user will now
behave exactly as the Looker API spec describes. The previous behavior
can still be accessed via sdk.auth.login, sdk.auth.logout, and
sdk.auth.login_user
```

I would have thought the `BREAKING CHANGE` footer text would be displayed here instead of the "body" of the commit

Reply:
Good reference. Interesting.